Sources/PackageLoading/PkgConfig.swift

```diff
@@ -123,7 +123,11 @@ public struct PkgConfig {
 
     private static var envSearchPaths: [AbsolutePath] {
         if let configPath = ProcessEnv.vars["PKG_CONFIG_PATH"] {
+#if os(Windows)
+            return configPath.split(separator: ";").map({ AbsolutePath(String($0)) })
```

Comment:
Would it make sense to factor this out into shared code?  If it's guaranteed to always use the same delimiter as `PATH`, then I think there's already code in UserToolchain that does this splitting.  Looks like there's something similar also in EnvironmentVariables.

In general I'm wary of oversharing of this kind of code (especially quoting, which really needs to be tailored to its use, e.g. quoting for a module map might not follow the same rules as for a shell, etc).  But if `pkg-config` already defines this to be the same as `PATH` delimiters, then it would make sense to share the implementation here, i.e. find a way to use the one in the UserToolchain.

Reply:
I think that sharing that does make sense.  I am slightly tempted to say that this is a good follow up change and should be separate from this change as this maintains the status quo structure.